make-util.js

```diff
@@ -160,13 +160,22 @@ var buildNodeTask = function (taskPath, outDir) {
         if (packageJson.devDependencies && Object.keys(packageJson.devDependencies).length != 0) {
             fail('The package.json should not contain dev dependencies. Move the dev dependencies into a package.json file under the Tests sub-folder. Offending package.json: ' + packageJsonPath);
         }
-
-        run('npm install');
+        if (process.env['BUILD_BUILDID']) {
```

Comment:
Can we add comments describing what this is checking?

Reply:
Sure, added